graphql-mode.el

```diff
@@ -62,8 +62,11 @@
 (defun graphql--query (query)
   "Send QUERY to the server at `graphql-url' and return the
 response from the server."
-  (let ((url-request-method "POST")
-        (url (format "%s?query=%s" graphql-url (url-encode-url query))))
+  (let* ((url-request-method "POST")
+	 (query (url-encode-url query))
+	 (operation (graphql-current-operation))
+	 (variables (url-encode-url (graphql-current-variables)))
+	 (url (format "%s?query=%s&operationName=%s&variables=%s" graphql-url query operation variables)))
```

Comment:
Can we add `operations` and `variables` as arguments, and delegate to `graphql-send-query` getting the actual values?

The idea was that `graphql--query` would work as a programatic way to create a query.  For example, we can use it to do introspection queries regardless of the buffer content.

Reply:
absolutely. I felt line 69 was too much as well.